proposals/0306-actors.md

```diff
@@ -542,15 +507,12 @@ extension BankAccount {
 }  
 ```
 
-Actors are similar to classes in all respects independent of isolation: actor types can have `static` and `class` methods, properties, and subscripts. All of the attributes that apply to classes apply to actors in much the same way, except where those semantics conflict with actor isolation.
```

Comment:
`static` stay though I hope? Just removal of the test here to compare with classes; no reason not to do static IMHO, very useful for IDs and stuff

Reply:
static is fine, I'll comment on it

Reply:
Might be good to clarify the isolation behavior (or not) of `static var`'s if they are allowed.  It might be worth banning them from actors?

Reply:
Sure, I can clarify that as well.

Reply:
Sounds good such properties to ban even, agreed :-)